Dockerfile.php5.6

```diff
@@ -95,3 +97,12 @@ RUN set -ex; \
 	{ \
 		echo "extension=newrelic.so"; \
 	} > /usr/local/etc/php/conf.d/docker-php-ext-newrelic.ini
+
+RUN set -ex; \
+	\
+	{ \
+		echo '[runkit]'; \
+	} >> /usr/local/etc/php/php.ini; \
+	{ \
+		echo "extension=$(find / -path /tmp -prune -o -name 'runkit.so' -print)"; \
```

Comment:
i believe these are zend_extensions.  extension/zend_extension also do not require their own INI section headers.